selfdrive/selfdrived/events.py

```diff
@@ -977,6 +977,14 @@ def personality_changed_alert(CP: car.CarParams, CS: car.CarState, sm: messaging
     ET.WARNING: personality_changed_alert,
   },
 
+  EventName.userFlag: {
+    ET.PERMANENT: Alert(
+      "Bookmark Saved",
+      "",
+      AlertStatus.normal, AlertSize.small,
+      Priority.LOW, VisualAlert.none, AudibleAlert.prompt, 1.5
+    ),
```

Comment:
let's just make this a NormalPermenantAlert, no need for a sound

Reply:
Sounds good